components/calendar_components/TaskList.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Task from "./Task";
+
+export default function TaskList({ tasks, toggleTask }) {
+  return (
+    <div className="task-list">
+      <div className="lefttasks">
+        {tasks.filter((task) => !task.complete).length} Aufgaben übrig
```

Comment:
I would probably add the filter as a variable before the return statement for better readabitlity:
```suggestion
  const uncompletedTasks = tasks.filter((task) => !task.complete).length

  return (
    <div className="task-list">
      <div className="lefttasks">
        {uncompletedTasks} Aufgaben übrig
```
Also, what happens if you don't have any tasks yet? And what is the default value if you don't have any `uncompleted` tasks?